mcc/odin/odin/mock_rds_client_test.go

```diff
@@ -16,6 +16,30 @@ type mockRDSClient struct {
 	dbSnapshots          []*rds.DBSnapshot
 }
 
+// AddSnapshots add a list of snapshots to the mock, both in the full
+// list and in the per instance map.
+func (m *mockRDSClient) AddSnapshots(
+	snapshots []*rds.DBSnapshot,
+) {
+	m.dbSnapshots = []*rds.DBSnapshot{}
+	for _, snapshot := range snapshots {
+		m.AddSnapshot(snapshot)
+	}
+}
+
+// AddSnapshot add a new snapshot to the mock, both in the full list
+// and the in the per instance map.
+func (m *mockRDSClient) AddSnapshot(
+	snapshot *rds.DBSnapshot,
```

Comment:
Completing what I explained in the previous comment, I just can add that `snapshot` here is an argument to the method... Might be the break lines are introducing more confusion? I use these for clarity, and to reduce the line length, but maybe I'm too obsessive about these...